pkg/tenant/topics.go

```diff
@@ -10,29 +10,68 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+// ProjectTopicList retrieves all topics assigned to a project
+// and returns a 200 OK response.
+//
+// Route: /projects/:projectID/topics
 func (s *Server) ProjectTopicList(c *gin.Context) {
-	c.JSON(http.StatusNotImplemented, "not implemented yet")
+	var (
+		err error
+	)
+
+	// Get the topic's project ID from the URL and return a 400 response
+	// if the project ID is not a ULID.
+	var projectID ulid.ULID
+	if projectID, err = ulid.Parse(c.Param("projectID")); err != nil {
+		log.Error().Err(err).Msg("could not parse project ulid")
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("could not parse project ulid"))
+		return
+	}
+
+	// Get topics from the database and return a 500 response
+	// if not successful.
+	if _, err = db.ListTopics(c.Request.Context(), projectID); err != nil {
```

Comment:
Looks like we will want to store the topics (`[]*Topic`) passed back by the database here so that we can assign the result to an `out` attribute